recipe/build.sh

```diff
@@ -1,6 +1,7 @@
 #!/bin/bash
 
-$PYTHON setup.py install --single-version-externally-managed --record record.txt
+# use the pip source to install itself
+PYTHONPATH="./src" $PYTHON -m pip install . --no-deps --no-build-isolation -vv
```

Comment:
These flags are [already set for us by `conda-build`]( https://github.com/conda/conda-build/blob/417d3f4c73ba525c037c1f453da24df8b1ee12aa/conda_build/build.py#L3052-L3072 )

```suggestion
PYTHONPATH="./src" $PYTHON -m pip install . -vv
```